android/src/main/java/com/waivpaykartasdk/AddToWallet.java

```diff
@@ -31,119 +33,141 @@
 import okhttp3.Response;
 
 public class AddToWallet {
-    public static final String HOST_STAGING = "https://webstores-staging.herokuapp.com/";
-    public static final String HOST_PRODUCTION = "https://webstore.egivv.com/";
+  public static final String HOST_STAGING = "https://webstores-staging.herokuapp.com/";
+  public static final String HOST_PRODUCTION = "https://webstore.egivv.com/";
+  private static final int REQUEST_CREATE_WALLET = 4;
+
+  public AddToWallet() {
+
+  }
```

Comment:
Is this right? It's an empty block

Reply:
yes, this is the constructor in Java; it had this one before. It's like the initial declaration.